<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakLayoutForm</class>
 <widget class="QWidget" name="BreakLayoutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,1,1" rowminimumheight="0,0,0,0">
     <item row="2" column="1">
      <widget class="QComboBox" name="cbTimeInterval"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Break Fill Method:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TV Program:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Break Time Interval:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cbProgram"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbBreakFillMethod"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lblEndTime">
       <property name="text">
        <string>End Time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lblStartTime">
       <property name="text">
        <string>Start time:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Week Days:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="cbMon">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTue">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWed">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbThu">
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbFri">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSat">
        <property name="text">
         <string>Saturday</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSun">
        <property name="text">
         <string>Sunday</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCopy">
     <property name="title">
      <string>Copy Hour</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item row="0" column="5">
       <widget class="QPushButton" name="btnUndo">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>From Hour:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cbToHour"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFromHour"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>To Hour:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnCopy">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="tbInsert">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDelete">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvBreakLayoutLine">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
